beacon-chain/sync-gossip.md

```diff
@@ -2,9 +2,9 @@
 >  This specification is a work-in-progress and should be considered preliminary.
 
 ## Overview
-A beacon chain client could sync committee to perform [state updates](https://github.com/ethereum/consensus-specs/blob/dev/specs/altair/sync-protocol.md). The data object [LightClientSnapshot](https://github.com/ethereum/consensus-specs/blob/dev/specs/altair/sync-protocol.md#lightclientsnapshot) allows a client to quickly sync to a particular header. Once the client establishes a valid header, it could sync to other headers by processing [LightClientUpdates](https://github.com/ethereum/consensus-specs/blob/dev/specs/altair/sync-protocol.md#lightclientupdate). These two data types allow a client to stay up-to-date with the beacon chain.
+A beacon chain client could sync committee to perform [state updates](https://github.com/ethereum/consensus-specs/blob/dev/specs/altair/sync-protocol.md). The data object [LightClientSkipSyncUpdate](skip-sync-network) allows a client to quickly sync to a reasonably recent header. Once the client establishes a recent header, it could sync to other headers by processing [LightClientUpdates](https://github.com/ethereum/consensus-specs/blob/dev/specs/altair/sync-protocol.md#lightclientupdate). These two data types allow a client to stay up-to-date with the beacon chain.
 
-These two data types are placed into separate gossip topics. A light client only needs to connects to `bc-light-client-snapshot` briefly at the start of the sync to get a reasonably recent beacon chain header. The client uses messages in the topic `bc-light-client-update` to advance its header.
+These two data types are placed into separate gossip topics. A light client make find-content requests on `skip-sync-network` at start of the sync to get a reasonably recent beacon chain header. The client uses messages in the gossip topic `bc-light-client-update` to advance its header.
```

Comment:
To get a `SyncCommittee` object in the current sync period

Reply:
updated.